test/Keywords.hs

```diff
@@ -49,25 +49,26 @@ propHandleControlString :: Property
 propHandleControlString = property $ testRoundTripValue $ VString "\x0011"
 
 propHandleUnicodeCharacters :: Property
-propHandleUnicodeCharacters = property $ for_ [minBound..maxBound] \c ->
+propHandleUnicodeCharacters = property $ liftTest $ for_ [minBound..maxBound] \c ->
```

Comment:
Small nit: we might want to add a note here that this is explicitly used to restrict the `for_` block to operate in the `Test` type (i.e. `type Test = TestT Identity`) which is a "thinner" monad stack & therefore doesn't suffer from as much memory leakage.

Reply:
Done.